src/main/java/net/countercraft/movecraft/warfare/events/SiegeBeginEvent.java

```diff
@@ -1,18 +1,51 @@
 package net.countercraft.movecraft.warfare.events;
 
+import net.countercraft.movecraft.localisation.I18nSupport;
 import net.countercraft.movecraft.warfare.siege.Siege;
+import org.bukkit.event.Cancellable;
 import org.bukkit.event.HandlerList;
 import org.jetbrains.annotations.NotNull;
 
-public class SiegeBeginEvent extends SiegeEvent {
+public class SiegeBeginEvent extends SiegeEvent implements Cancellable {
```

Comment:
It might be a good idea to throw a comment here to describe when SiegeBeginEvent and SiegeStartEvent are called.